contracts/protocol/v1/proxies/P1SoloBridgeProxy.sol

```diff
@@ -320,13 +357,16 @@ contract P1SoloBridgeProxy is
             otherAccountId: 0,
             data: ""
         });
+        if (isWithdrawal) {
+            soloActions[0].actionType = I_Solo.ActionType.Withdraw;
+        }
```

Comment:
nit: I think and in-line ternary would be clearer than overwriting `soloActions[0].actionType` here.

e.g.
```
actionType: isWithdrawal ? I_Solo.ActionType.Withdraw : I_Solo.ActionType.Deposit,
```